docs/nodes.rst

```diff
@@ -1,15 +1,8 @@
 Nodes
 ------------------
 
-.. toctree::
-    :maxdepth: 2
+.. autosummary::
+     :toctree: stubs
 
-.. autoclass:: tensornetwork.Node
-    :members:
-
-.. autoclass:: tensornetwork.CopyNode
-    :members:
-
-.. autoclass:: tensornetwork.BaseNode
-    :members:
```

Comment:
Don't forget to include `CopyNode` and `BaseNode`.

Reply:
Done, sorry I missed it first time for some reason :<